en/docs/fragments/saml-config-sample.md

```diff
@@ -0,0 +1,18 @@
+1.	Expand **Inbound Authentication Configuration > SAML Configuration** and click **Configure**.
+
+2.	Enter the following details and keep the values of the rest of the fields as it is. 
+
+	-	**Issuer**:simplesaml
```

Comment:
since this config is specific for `simplesaml` let's rename the file to match this.